### PR TITLE
fixed markdown examples and added OPS extensions

### PR DESCRIPTION
**Pull request by sdwheeler**
Fixed Markdown examples for proper indenting. Added Markdown Reference article. The how to article references use of OPS extensions but does not show the syntax. The Markdown Reference article shows the syntax.

### CODE REVIEW

**Comment by JasonWHowell** [comment]
@BillWagner if this is merged, why did the PR still stay open?

**Comment by BillWagner** [comment]
@sdwheeler @JasonWHowell This PR got caught in the GitHub service outage.

The changes were merged. but the PR status wasn't updated.

I'll try to close by hand.

File: Contribute/git-github-fundamentals.md [hunk 1]
```diff
@@ -40,7 +40,13 @@ All workflows begin and end at the GitHub level, where the main repository for a
...
 
-You can typically find a main `articles` directory off the root of the repository. The articles directory contains a set of subdirectories. Articles in the subdirectories are formatted as Markdown files that use an *.md* extension. Some repositories that support multiple services use a generic `/articles` subdirectory, such as the [https://github.com/microsoft/Azure-Docs](https://github.com/microsoft/Azure-Docs) repository. Others might use a service-specific name, such as the [https://github.com/microsoft/IntuneDocs](https://github.com/microsoft/IntuneDocs) repository, which uses `/IntuneDocs`.
+You can typically find a main `articles` directory off the root of the repository. The articles
+directory contains a set of subdirectories. Articles in the subdirectories are formatted as
+Markdown files that use an *.md* extension. Some repositories that support multiple services use a
+generic `/articles` subdirectory, such as the
+[Azure-Docs](https://github.com/MicrosoftDocs/Azure-Docs) repository. Others might use a
```
Comment (JasonWHowell) [inline Contribute/git-github-fundamentals.md]: Something is going on with word wrap here.

**Comment by JasonWHowell** [comment]
@BillWagner It doesn't show up yet. https://docs.microsoft.com/en-us/contribute/markdown-reference 
The new file isn't present yet: https://github.com/MicrosoftDocs/Contribute/tree/master/Contribute

**Comment by BillWagner** [comment]
Thanks @JasonWHowell Re-opening.

**Pull request merged**